package.nls.json

```diff
@@ -188,6 +188,8 @@
     "vscode-docker.commands.containers.start": "Start",
     "vscode-docker.commands.containers.stop": "Stop",
     "vscode-docker.commands.containers.viewLogs": "View Logs",
+    "vscode-docker.commands.containers.composeGroup.restart": "Compose Restart",
```

Comment:
TODO: these strings good?

Reply:
Good enough IMO ☺️

Reply:
@BigMorty What do you think? These strings are what show in the context menu.